MapboxNavigation/NavigationMapView.swift

```diff
@@ -1101,20 +1097,28 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     /**
      Sets the camera directly over a series of coordinates.
      */
-    @objc public func setOverheadCameraView(from userLocation: CLLocationCoordinate2D, along coordinates: [CLLocationCoordinate2D], for bounds: UIEdgeInsets) {
-        let slicedLine = Polyline(coordinates).sliced(from: userLocation).coordinates
-        let line = MGLPolyline(coordinates: slicedLine, count: UInt(slicedLine.count))
-        
+    @objc public func setOverheadCameraView(from userLocation: CLLocationCoordinate2D, along coordinates: [CLLocationCoordinate2D], edgePadding: UIEdgeInsets) {
         tracksUserCourse = false
-        let camera = self.camera
-        camera.pitch = 0
-        camera.heading = 0
-        self.camera = camera
-        
-        // Don't keep zooming in
-        guard line.overlayBounds.ne.distance(to: line.overlayBounds.sw) > NavigationMapViewMinimumDistanceForOverheadZooming else { return }
+        let slicedLine = Polyline(coordinates).sliced(from: userLocation).coordinates
+        navigationCamera.transitionToOverview(coordinates: slicedLine, edgePadding: edgePadding)
+    }
+    
+    func updateAltitude(_ altitude: CLLocationDistance) {
```

Comment:
Would you mind exposing this method as `public` ?!
I would like to update the camera `altitude` from the outside, e.g. using the `NavigationMapView` as a single entity.